go.mod

```diff
@@ -3,20 +3,20 @@ module github.com/crossplane/templating-controller
 go 1.13
 
 require (
-	github.com/crossplane/crossplane v0.8.0-rc.0.20200219030944-557074689d71
-	github.com/crossplane/crossplane-runtime v0.5.1-0.20200219022054-8948e50564b1
+	github.com/crossplane/crossplane v0.10.0-rc.0.20200429060016-7ced24cf6bfa
+	github.com/crossplane/crossplane-runtime v0.7.1-0.20200429054236-05ca13f8ea7b
 	github.com/google/go-cmp v0.4.0
 	github.com/pkg/errors v0.9.1
-	golang.org/x/net v0.0.0-20191028085509-fe3aa8a45271
+	golang.org/x/net v0.0.0-20200202094626-16171245cfb2
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6
-	helm.sh/helm/v3 v3.0.0-20200205083830-5ec70ab27fbf
-	k8s.io/api v0.17.2
-	k8s.io/apiextensions-apiserver v0.17.2
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
-	sigs.k8s.io/controller-runtime v0.4.0
+	helm.sh/helm/v3 v3.2.0
+	k8s.io/api v0.18.2
+	k8s.io/apiextensions-apiserver v0.18.2
+	k8s.io/apimachinery v0.18.2
+	k8s.io/client-go v0.18.2
+	sigs.k8s.io/controller-runtime v0.6.0
 	sigs.k8s.io/kustomize/api v0.3.0
-	sigs.k8s.io/yaml v1.1.0
+	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
```

Comment:
FWIW, it seems to be safe to remove the last `blackfriday` replace statement as Helm 3.2.0 doesn't use replace for that dependency [anymore](https://github.com/helm/helm/blob/release-3.2/go.mod).

Reply:
Nice, I'll remove!